plugins/baseball-scoreboard/milb_managers.py

```
@@ -0,0 +1,390 @@
+            response = self.session.get(
+                url, headers=self.headers, timeout=30
+            )
+            response.raise_for_status()
+            data = response.json()
+
+            for date_entry in data.get("dates", []):
+                for game in date_entry.get("games", []):
+                    event = self._convert_stats_game_to_espn_event(game)
+                    all_events.append(event)
+
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)